bridge.go

```diff
@@ -17,6 +17,12 @@ type Bridge struct {
 	ID   string `json:"id,omitempty"`
 }
 
+// CreatedUser represents a created user from the Configuration API
+type CreatedUser struct {
+	Username  string `json:"username,omitempty"`
+	ClientKey string `json:"clientkey,omitempty"`
+}
```

Comment:
No need for this struct. We can simply put `ClientKey` as a field to the existing `Whitelist` struct.

Reply:
I'll take a look at that struct, I believe you are correct